docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -21,168 +21,109 @@ helpviewer_keywords:
+| Naming style | `dotnet_naming_style` | `dotnet_naming_style.pascal_case.capitalization = pascal_case` |
+| Naming rule | `dotnet_naming_rule` | `dotnet_naming_rule.types_should_be_pascal_case.severity = suggestion`
 
-A naming rule matches signatures that have *all* the modifiers specified in `required_modifiers`. If you omit this property, the default value of an empty list is used, that is, no specific modifiers are required for a match. This means a symbol's modifiers have no effect on whether or not this rule is applied.
+Each type of definition -- [naming rule](#naming-rule-properties), [symbol group](#symbol-group-properties), or [naming style](#style-properties)  -- has its own supported properties, as described below.
 
-> [!TIP]
-> Do not specify a value of `*` for `required_modifiers`. Instead, just omit the `required_modifiers` property altogether and your naming rule will apply to any kind of modifier.
+### \<title>
 
-## Style
+**\<title>** is a descriptive name that you choose, which associates multiple property settings into a single definition. For example, the following:
 
-Now that you've identified the group of symbols to apply the naming rule to, you can describe the naming style. A style can be that the name has a certain prefix or a certain suffix, or that individual words in the name are separated with a certain character. You can also specify a capitalization style. The style property has the following format:
-
-`dotnet_naming_rule.<namingRuleTitle>.style = <styleTitle>`
-
-Give the style a name by replacing the **<styleTitle\>** value with a descriptive title, for example `first_word_upper_case_style`. You'll use the **<styleTitle\>** value in the property names that describe the naming style (prefix, suffix, word separator character, and capitalization). Use one or more of these properties to describe your style.
+```ini
+dotnet_naming_symbols.interface.applicable_kinds = interface
+dotnet_naming_symbols.interface.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
 
-### Require a prefix
+dotnet_naming_symbols.types.applicable_kinds = class, struct, interface, enum, delegate
+dotnet_naming_symbols.types.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+```
 
-To specify that symbol names must begin with certain characters, use this property:
+produces two symbol group definitions -- `interface` and `types` -- each of which has two properties set on it.
 
-`dotnet_naming_style.<styleTitle>.required_prefix = <prefix>`
+## Naming rule properties
 
-### Require a suffix
+All naming rule properties are required.
 
-To specify that symbol names must end with certain characters, use this property:
+| Property | Description |
+| -- | -- |
+| `symbols` | The title of the symbol group, defining the symbols to which this rule should be applied |
```

Comment:
```suggestion
| One of the [symbol group properties](#symbol-group-properties) | Defines the symbols to which this rule should be applied, for example, kind, accessibility, and modifiers. |
```

Reply:
This isn't correct. We're listing the properties which can be set on a naming rule definition; `symbols` is one of those properties, and the value will be the title of one of the symbol groups defined elsewhere. This is entirely distinct from the properties that can set on a symbol group.